docs/config.md

```diff
@@ -608,10 +608,10 @@ profile_models:
       dynamic_batching:
         max_queue_delay_microseconds: [200, 400, 600]
       instance_group:
-        - - kind: KIND_GPU
-            count: 1
-        - - kind: KIND_CPU
-            count: 1
+        - kind: KIND_GPU
```

Comment:
Are you sure these changes are correct?

Reply:
I think with single dash it would be [GPU, CPU], which would tell MA to put one instance on a GPU and one instance on a CPU. 
I think with double dash it would be [GPU],[CPU], which would tell MA to try GPU and then try CPU. Double dash might be correct here.

Reply:
We should test to confirm, and make sure it is correct/consistent in this file